src/lib/entry.js

```diff
@@ -5,7 +5,8 @@ if (process.env.NODE_ENV==='development') {
 	require('preact/devtools');
 }
 else if ('serviceWorker' in navigator && location.protocol === 'https:') {
-	navigator.serviceWorker.register('/sw.js');
+	// eslint-disable-next-line no-undef
+	navigator.serviceWorker.register(__webpack_public_path__ + 'sw.js');
```

Comment:
@developit Please take a look at this and decide finally about the solution. I really need this feature in my project :-) 
In my opinion it is a nice solution, what are your concerns?

Reply:
I'm fine with this, webpack will never change this var name and it keeps the resolution at runtime.